src/Nethermind/Nethermind.Serialization.Ssz/Ssz.ShardBlobTransaction.cs

```diff
@@ -0,0 +1,322 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Eip2930;
+using Nethermind.Int256;
+
+namespace Nethermind.Serialization.Ssz;
+
+public static partial class Ssz
+{
+    public static int BlobTransactionNetworkWrapperLength(Transaction container)
+    {
+        ShardBlobNetworkWrapper? wrapper = container.NetworkWrapper as ShardBlobNetworkWrapper;
+        if (wrapper is null)
+        {
+            throw new ArgumentException("Wrapper should present in network form", nameof(container.NetworkWrapper));
+        }
+
+        int transactionDynamicOffset = 4 * VarOffsetSize;
+        int commitmentsDynamicOffset = transactionDynamicOffset + SignedBlobTransactionLength(container);
+        int blobsDynamicOffset =
+            commitmentsDynamicOffset + wrapper.Commitments.Length;
+        int proofsDynamicOffset = blobsDynamicOffset + wrapper.Blobs.Length;
+
+        return proofsDynamicOffset + wrapper.Proofs.Length;
```

Comment:
Looks overcomplicated, I would go with something more readable, like: 
`int transactionDynamicOffset = 4 * VarOffsetSize;`
`transactionDynamicOffset += SignedBlobTransactionLength(container);`
`transactionDynamicOffset += wrapper.Commitments.Length;`
`transactionDynamicOffset += wrapper.Blobs.Length;`
`transactionDynamicOffset += wrapper.Proofs.Length;`
`return transactionDynamicOffset;`
But it is just readability, not logic, feel free to ignore

Reply:
Made it even cleaner, thanks